tests/epics/adkinetix/test_kinetix.py

```diff
@@ -1,38 +1,28 @@
 import pytest
-from bluesky.run_engine import RunEngine
 
 from ophyd_async.core import (
     DetectorTrigger,
-    DeviceCollector,
     StaticPathProvider,
     set_mock_value,
 )
 from ophyd_async.core._detector import TriggerInfo
-from ophyd_async.epics import adkinetix
+from ophyd_async.epics.adkinetix import KinetixDetector
```

Comment:
We actually imported `adkinetix` rather than `from adkinetix import KinetixDetector` as that is the pattern we are about to recommend in all the tutorials, so we updated all the tests to follow that pattern